vmdb/spec/automation/unit/method_validation/orchestration_provision_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+describe "Orchestration provision Method Validation" do
```

Comment:
This is not a spec for class ServiceOrchestration. I am following the convention to use "Method Validation" in other tests in this folder.